source/statements.tex

```diff
@@ -245,6 +245,50 @@
 an \tcode{else} part.\footnote{In other words, the \tcode{else} is associated with the nearest un-elsed
 \tcode{if}.}
 
+\pnum
+If the parenthesized \grammarterm{condition} is prefixed with
+\tcode{constexpr}, the value of the condition shall be a contextually
+converted constant expression of type bool (\ref{expr.const}); this
+form is called a \term{constexpr if} statement. If the value of the
+converted condition is \tcode{false}, the first substatement is a
+\term{discarded statement}, otherwise the second substatement, if
+present, is a discarded statement. During the instantation of an
+enclosing templated entity, if the condition is not value-dependent
```

Comment:
Do you think "templated entity" should come with a cross reference to "Clause 14"?